tests/ref/simulation/service_init.json

```diff
@@ -70,7 +70,7 @@
         "absolute_tolerance": 1e-10,
         "relative_tolerance": 0.0,
         "order_of_accuracy": 2,
-        "equation_eval_frequency": 1,
+        "equation_evaluation_frequency": 1,
```

Comment:
why do we change to full name? If so should we enforce full name throughout? like 
`max_force_jac_update_physical_steps` --> `max_force_jacobian_update_physical_steps` ?

Reply:
Yeah I think ideally we should enforce full name throughout. Particularly I didn't think eval short for evaluation was a common use